ansi.sublime-settings

```diff
@@ -39,5 +39,7 @@
     "invisibles":       "#FFFFFF00",
     "lineHighlight":    "#FFFFFF00",
     "selection":        "#FFFFFF33"
-  }
+  },
+  "process_on_finish": true,
+  "process_on_data": false,
```

Comment:
Maybe this would be clearer as a single option, for example:
"process_build" : "on_output" or "on_complete"